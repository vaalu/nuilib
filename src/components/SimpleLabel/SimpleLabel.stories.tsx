import React from 'react'
import { storiesOf } from '@storybook/react'
import { SimpleLabel } from './SimpleLabel'
storiesOf('Atom/SimpleLabel', module)
	.add('Label-1', () => <SimpleLabel title='Label 1' />)
