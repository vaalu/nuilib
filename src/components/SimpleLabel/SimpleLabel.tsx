import React from 'react'
export type DefaultLabel = {
	title: string
}
export const SimpleLabel = (labelParams:DefaultLabel) => {
	const {title} = labelParams
	return(<div>
		This is a Simple Label: { title }
	</div>)
}
