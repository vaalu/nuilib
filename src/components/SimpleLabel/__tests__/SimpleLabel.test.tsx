import TestRenderer from 'react-test-renderer'
import React from 'react'
import { SimpleLabel } from '../SimpleLabel'
describe('Simple Label Tests', () => {
	it('Simple Label T1', () => {
		const tree = TestRenderer.create(
            <SimpleLabel title='Simple Label' />
		).toJSON()
		expect(tree).toMatchSnapshot()
	})
})