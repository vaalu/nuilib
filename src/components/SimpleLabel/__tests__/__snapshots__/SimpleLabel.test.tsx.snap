// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Simple Label Tests Simple Label T1 1`] = `
<div>
  This is a Simple Label: 
  Simple Label
</div>
`;
